tests/e2e/ctl_v3_member_test.go

```diff
@@ -55,6 +59,91 @@ func TestCtlV3MemberUpdatePeerTLS(t *testing.T) {
 	testCtl(t, memberUpdateTest, withCfg(*e2e.NewConfigPeerTLS()))
 }
 
+// TestCtlV3ConsistentMemberList requires the gofailpoint to be enabled.
+// If you execute this case locally, please do not forget to execute
+// `make gofail-enable`.
```

Comment:
From high level propestive, gracefully shutting down means letting the process to do some cleanup tasks,  including syncing the data. I think it's simple to understand.

From implementation perspective, it needs deep understanding on the repo. It's not about one line of code, it's about a complicated workflow.

For anyone reference: 
- When the etcd receives SIGTERM, it will trigger (*Etcd)Close https://github.com/etcd-io/etcd/blob/0a3dc1a8a8b6d06368ad13f9a8f13c038e7ca362/server/embed/etcd.go#L388

- Inside the Etcd.Close, it will execute EtcdServer.HardStop https://github.com/etcd-io/etcd/blob/0a3dc1a8a8b6d06368ad13f9a8f13c038e7ca362/server/etcdserver/server.go#L1244

- The main goroutine will exit at https://github.com/etcd-io/etcd/blob/0a3dc1a8a8b6d06368ad13f9a8f13c038e7ca362/server/etcdserver/server.go#L854

- Then the defered function will be executed, inside which s.Cleanup will be executed. https://github.com/etcd-io/etcd/blob/0a3dc1a8a8b6d06368ad13f9a8f13c038e7ca362/server/etcdserver/server.go#L829

- Inside the cleanup, backend.Close is executed. https://github.com/etcd-io/etcd/blob/0a3dc1a8a8b6d06368ad13f9a8f13c038e7ca362/server/etcdserver/server.go#L910

- Eventually it will execute bbolt.Commit https://github.com/etcd-io/etcd/blob/0a3dc1a8a8b6d06368ad13f9a8f13c038e7ca362/server/storage/backend/backend.go#L421